client/js/components/snippets/snippet.jsx

```diff
@@ -124,22 +130,25 @@ export default class Snippet extends React.Component {
       mime: this.props.language
     };
     let { style } = this.props;
-    let author = this.props.user.name;
-
+    let currentUser = this.getCurrentUser();
+    let author = this.props.user || currentUser;
+    let authorName = (author && author.name) || 'author';
```

Comment:
i case you're asking about lack of props.user && current user , it happens sometimes when session didn't reload good and current user is null , so i added this one for safety @vrael560